starfish/util/try_import.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+from typing import Any
+
+def try_import(module: str) -> Any:
```

Comment:
I have some concerns about this approach given that it makes IDEs and mypy probably very unhappy, so check out #734.